packages/paste-website/src/pages/layout/stack/index.mdx

```diff
@@ -0,0 +1,163 @@
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/stack/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Stack"
+  categoryRoute={SidebarCategoryRoutes.LAYOUT}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/layout/stack"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/layout-stack--stack"
+  data={props.data.allPasteLayout.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Stack
+
+Stack is a layout component that allows you to easily stack elements together with a space between them.
+Stack can be set up either vertically or horizontally, and uses tokens for spacing. The spacing tokens
+are used as either a margin-right for horizontal, or a margin-bottom for vertical.
+
+#### Accessibility
+
+The Stack component is an all purpose component. By default, it has no accessibility concerns. If you use
+the Stack as a custom element, it is up to you to manage the resulting accessibility implications.
+
+## Examples
+
+### Vertical Stack
+
+Elements within the stack are stacked vertically with a bottom margin. This is achieved by setting
+`orientation="vertical"`. A `spacing` property is provided to let you to create space between each stacked element.
+`spacing` will take any [Spacing Token](/tokens#spacings) as its value.
+
+<LivePreview scope={{Button, Card, Heading, Paragraph, Stack}} language="jsx">
+  {`<Stack orientation="vertical" spacing="space60">
+  <Card>
+    <Heading as="h3" variant="heading30">What's new at Twilio</Heading>
+    <Paragraph>Check out our newest product releases on the Twilio Changelog.</Paragraph>
+    <Button variant="primary">Changelog</Button>
+  </Card>
+  <Card>
+    <Heading as="h3" variant="heading30">Twilio Docs</Heading>
+    <Paragraph>Dig into our API reference documentation and quickstarts. You're covered with everything from SDKs in your favorite languages to sample apps for web, iOS, and Android.</Paragraph>
+    <Button variant="secondary">Explore the Docs</Button>
+  </Card>
+</Stack>`}
+</LivePreview>
+
+### Horizontal Stack
+
+Elements within the stack are stacked horizontally with a right margin. This is achieved by setting
+`orientation="horizontal"`. A `spacing` property is provided to let you to create space between each stacked element.
+`spacing` will take any [Spacing Token](/tokens#spacings) as its value.
+
+<LivePreview scope={{Anchor, Button, Stack}} language="jsx">
+  {`<Stack orientation="horizontal" spacing="space60">
+  <Button variant="primary">Save</Button>
+  <Button variant="secondary">Cancel</Button>
+  <Anchor href="#">Close Project</Anchor>
+</Stack>`}
+</LivePreview>
+
+### Responsive Stack
+
+The stack flow can change based on your theme breakpoints, so a horizontal stack could be vertical on
```

Comment:
This paragraph is not very clear, can you simplify it?

Reply:
Dropped the first part.